OPA.ps1

```diff
@@ -19,21 +19,17 @@ $OPAExe = "opa_windows_amd64.exe"
 $InstallUrl = "https://openpolicyagent.org/downloads/v$($ExpectedVersion)/$OPAExe"
 $OutFile=(Join-Path (Get-Location).Path $InstallUrl.SubString($InstallUrl.LastIndexOf('/')))
 $ExpectedHash ="5D71028FED935DC98B9D69369D42D2C03CE84A7720D61ED777E10AAE7528F399"
-
+$Display = "Downloading OPA executable"
 # Download files
 try {
-    Write-Information "Downloading $InstallUrl"
-    $WebClient = New-Object System.Net.WebClient
-    $WebClient.DownloadFile($InstallUrl, $OutFile)
+    Start-BitsTransfer -Source $InstallUrl -Destination $OutFile -DisplayName $Display
```

Comment:
Yea, i didnt think of that, BITS will not work on non-Windows systems. i will explore other ways to download the file. with progress bar.

Reply:
so currently the download is within seconds. there is another script that should work for both windows and non-windows. However, it will slow down to download to about 1 minute n 20 seconds. So do we want more versatility or speed?